### ISSUE TITLE
Django Middleware

### ISSUE THREAD

**Issue by mwoya** [mbrochh/django-graphql-apollo-react-demo#7]
Following your code, I got some error when implementing JWT middleware.
after inserting 

backend.middleware.JWTMiddleware',

The issue is my middleware is
MIDDLEWARE = [
[...]
    'backend.middleware.JWTMiddleware',
[...]

I found that **MIDDLEWARE** is the newer version and is not compatible with your code.
the solution is to change my setting into **MIDDLEWARE_CLASSES**.

I'm new in programming, I don't know if this information is useful.

**Comment by mbrochh** [comment]
Good point! I will soon run this whole tutorial again. I can only imagine that there was a new Django version released in the meantime and that's why the name of the setting has now changed. I will update the tutorial so that it uses pinned versions for everything to avoid such conflicts in the future.
